app/Models/Country.php

```diff
@@ -2,14 +2,25 @@
 
 namespace App\Models;
 
+use App\Traits\BaseModel;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 
 class Country extends Model
 {
-    use HasFactory;
+    use BaseModel,HasFactory;
+
+    public $queryable = [
+        'id',
+    ];
 
     protected $fillable = [
-        'country_code',
+        'iso',
+        'name',
+        'iso3',
+        'numcode',
+        'phonecode',
     ];
+
+    protected $relationship = [];
```

Comment:
@dhrumin-7span Please remove the empty relationship varibale

Reply:
Removed.